File: autofin/src/main/java/model/vehicle_details/vehicle_category/stock_details/StockDetailsReq.java
package model.vehicle_details.vehicle_category.stock_details;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class StockDetailsReq {
    @SerializedName("UserId")
    @Expose
    private String userId;
    @SerializedName("UserType")
    @Expose
    private String userType;
    @SerializedName("Data")
    @Expose
    private StockData data;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUserType() {
        return userType;
    }

    public void setUserType(String userType) {
        this.userType = userType;
    }

    public StockData getData() {
        return data;
    }

    public void setData(StockData data) {
        this.data = data;
    }

}

File: autofin/src/main/java/v2/model/request/StockDetailsReq.kt
package v2.model.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import v2.model.request.add_lead.AddLeadData

public class StockDetailsReq {

    @SerializedName("userId")
    public var UserId: String? = null
    @SerializedName("userType")
    public var UserType: String? = null
    @SerializedName("requestFrom")
    public var RequestFrom: String? = null
    @SerializedName("data")
    var data : VehicleRegNum? = null
    @SerializedName("vehicleNumber")
    public var vehicleNumberE : String? =null
    @SerializedName("accessKey")
    public var accessKey : String? =null

}